chapter_convolutional-modern/batch-norm.md

```diff
@@ -285,6 +285,18 @@ def batch_norm(X, gamma, beta, moving_mean, moving_var, eps, momentum):
     return Y, moving_mean, moving_var
 ```
 
+```{.python .input}
+#@tab tensorflow
+from d2l import tensorflow as d2l
+import tensorflow as tf
+
+# TODO: Check whether it's necessary to re-implement from scatch here.
```

Comment:
@mli What do you think? I feel like it's unnecessary to re-implement from scratch since we won't be reusing it in other sections anyways.